dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/data/upload/UploadStatus.kt

```diff
@@ -117,6 +117,8 @@ internal sealed class UploadStatus(
 
             if (throwable != null) {
                 append(" (")
+                append(throwable.javaClass.name)
```

Comment:
the downside is that it will be obfuscated in the release builds, so it will come as obfuscated to our telemetry as well. But I guess the major use-case is still debugging the issue with the debug build.

Reply:
It will only be obfuscated if it's a user or 3rd party lib exception. All the Java and Android framework excpetions will be in cleartext (the support case that sprung this was because of an ArrayIndexOutOfBounds issue)